ProjectSidewalk/SidewalkWebpage
# Resolves #2961: Hide/show label button on Validate missing translations

Resolves #2961
- Hide/show label button on Validate missing translations

##### Before/After screenshots (if applicable)
Before:
![image](https://user-images.githubusercontent.com/78244145/187129925-9c954321-f0e0-41c8-a31f-fe3812538721.png)
After:
![218341661751198_ pic](https://user-images.githubusercontent.com/78244145/187129822-f748722f-cd02-440f-930d-0573d7d0954c.jpg)
![218351661751198_ pic](https://user-images.githubusercontent.com/78244145/187129831-10f82ab5-8a9c-4dc8-8015-d3dff22eff0d.jpg)

##### Testing instructions
1. Switch es or nl language
2. Click 'Start Validating'
3. Switch Hide/Show label and check whether it is correct according to the language type.
